docs/podman-remote.1.md

```diff
@@ -1,7 +1,7 @@
 % podman-remote(1)
 
 ## NAME
-podman-remote - A remote CLI for Podman: A Simple management tool for pods, containers and images.
+podman-remote - A tool for remote management of pods, containers and images
```

Comment:
I'm not in favor of this change.  I'll let @baude and @jwhonce chime in.

Reply:
I'm trying to find a way to exclude podman-remote from the hack/man-page-checker description check; I'll revert this change.

Reply:
for md in $(ls -1 *-*.1.md | grep -v remote);do